ghu_web/ghu_main/static/ghu_main/css/base.css

```diff
@@ -54,3 +58,32 @@
     background: #4aaec6; /* Fallback */
     background: linear-gradient(#4aaec6, #2dc5ea);
 }
+
+/* Mobile Size */
+@media (max-width: 770px ) {
+    #logo {
+        margin-top: 18px;
+        margin-bottom: 18px;
+    }
+
+    #logo a {
+        font-size: 5.2vw;
+    }
+
+    #jumbotron-title {
+        margin-top: 210px;
+    }
+
+    #jumbotron-title p {
+        margin-left: 10px;
+        padding-left: 30px;
+        font-size: 3.7vw;
```

Comment:
I like this idea, but I'm not sure how well it would work. Because this ends up making the text in `#jumbotron-title p` unreadably small in narrow viewports (see image above), I think we should just go ahead and `display: none` it here instead

Reply:
yea ill change it to display: none